projects/rocprofiler-sdk/source/include/rocprofiler-sdk/context.h

```diff
@@ -24,6 +24,7 @@
 
 #include <rocprofiler-sdk/defines.h>
 #include <rocprofiler-sdk/fwd.h>
+#include <rocprofiler-sdk/registration.h>
```

Comment:
I'd rather move `rocprofiler_context_id_t` into fwd.h instead of including registration.h here